network/src/peers/peer_info.rs

```diff
@@ -183,21 +207,10 @@ impl PeerInfo {
         self.quality.expecting_pong.store(false, Ordering::SeqCst);
         self.quality.remaining_sync_blocks.store(0, Ordering::SeqCst);
 
-        for (handle, abortable) in self.tasks.lock().drain(..).rev() {
-            if abortable {
-                handle.abort();
-            } else {
-                task::spawn(async move {
-                    // An arbitrary amount of time to allow the task to shut down cleanly.
-                    if tokio::time::timeout(Duration::from_secs(5), handle).await.is_err() {
```

Comment:
we need to maintain this functionality

Reply:
what is the purpose of this functionality? i would expect an immediate termination to be the expected behavior. is there a concern with not flushing packets to network fully, or something to that effect?

Reply:
This is the same thing as described in https://github.com/AleoHQ/snarkOS/pull/725#discussion_r637777974.